dev/com.ibm.ws.container.service/src/com/ibm/ws/container/service/annocache/Annotations.java

```diff
@@ -0,0 +1,398 @@
+package com.ibm.ws.container.service.annocache;
```

Comment:
No copyright

Reply:
Fixed.  (Also, reviewed the others under container/service/annocache.